src/raster/r.survey/r.survey.bat

```diff
@@ -0,0 +1 @@
+@"%GRASS_PYTHON%" "%GISBASE%/scripts/r.survey.py" %*
```

Comment:
@IvanMarchesini May this file also be deleted? Since it will become built for Windows automatically, I don't see any purpose in keeping this .bat file.

Reply:
For sure it can be removed.. 
just did it (I hope)